test/models/normalize_worldcat_test.rb

```diff
@@ -59,4 +59,16 @@ class NormalizeWorldcatTest < ActiveSupport::TestCase
       assert_equal(0, query['total'])
     end
   end
+
+  test 'searches with many authors get truncated list' do
```

Comment:
Also not really necessary (see comment about test in `test/models/normalize_eds_test.rb`)